packages/devextreme/js/__internal/grids/grid_core/__test__/utils.test.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line forbidden-imports/no-restricted-imports
+import { popupUtils } from '../m_utils';
+
+describe('utils', () => {
```

Comment:
Let's refactor these test cases a little and add the next scenarios:

- let's check all properties, not only the ```shortcut``` (I mean the ```toolbar``` and ```location``` too)
May be it will be good idea to match the ```result``` with the ```expectedResult``` object
- let's check that the ```option``` argument passed to these functions will exist in the ```result```